doc/command_ref.rst

```diff
@@ -926,6 +926,13 @@ Module subcommands take :ref:`\<module-spec>\ <specifying_modules-label>`... arg
     required by other installed module profiles or by other user-installed packages.
     In case no profile was provided, all installed profiles get removed.
 
+``dnf [options] module remove --all <module:stream>...``
```

Comment:
Why didn't you use `--all <module-spec>` to make it consistent with the rest of the documentation?

Reply:
The command prefers <module:stream> format. If you will add additional information like version, or context, you will get warning, but I can change it